### ISSUE TITLE
Can not reproduce dino with R101 backbone

### ISSUE THREAD

**Issue by adamqian111** [IDEA-Research/detrex#118]
hello, I cant reproduce dino with R101 backbone. I only got 47.15 total AP without same config.

https://github.com/IDEA-Research/detrex/blob/main/projects/dino/configs/dino_r101_4scale_12ep.py

**Comment by rentainhe** [comment]
> hello, I cant reproduce dino with R101 backbone. I only got 47.15 total AP without same config.
> 
> https://github.com/IDEA-Research/detrex/blob/main/projects/dino/configs/dino_r101_4scale_12ep.py

Would you like to share your training log with us ?

For resnet-101 model, you need to convert the torchvision weights first and then set the initialized weight path as `train.init_checkpoint=/path/to/r101.pkl`

**Comment by rentainhe** [comment]
> hello, I cant reproduce dino with R101 backbone. I only got 47.15 total AP without same config.
> 
> https://github.com/IDEA-Research/detrex/blob/main/projects/dino/configs/dino_r101_4scale_12ep.py

You can refer to this [tutorial](https://detrex.readthedocs.io/en/latest/tutorials/Converters.html#download-pretrained-weights) to see how to convert the R-101 pretrained weights from torchvision downloaded checkpoints.

**Comment by adamqian111** [comment]
Thanks, Maybe I dont convert torchvision model to d2 model.
thanks again. I will close this issue.

**Comment by rentainhe** [comment]
> Thanks, Maybe I dont convert torchvision model to d2 model. thanks again. I will close this issue.

You're welcome, feel free to reopen it if you meet some other problems, You can also try to set the training seed to 42
```python
# your config
train.seed = 42
```
which aligns the official training settings, this may help you to reproduce the 49.95 AP results

**Comment by backseason** [comment]
> > Thanks, Maybe I dont convert torchvision model to d2 model. thanks again. I will close this issue.
> 
> You're welcome, feel free to reopen it if you meet some other problems, You can also try to set the training seed to 42
> 
> ```python
> # your config
> train.seed = 42
> ```
> 
> which aligns the official training settings, this may help you to reproduce the 49.95 AP results @adamqian111

Hello, after fixing the random seed to 42, I still can not reproduce the results of dino_r50_4scale_12ep which is 49.2 in your repo. Do you have any ideas where the problem might bt?
<img width="813" alt="image" src="https://user-images.githubusercontent.com/7400760/202632285-9ff718da-f69b-46b0-bb13-cdebd585b4b8.png">

**Comment by cyoukaikai** [comment]
Probably you are loading R-50.pkl (From your log file,[log_detrex_dino_r50_4scale_12ep_48.87.txt](https://github.com/IDEA-Research/detrex/files/10038625/log_detrex_dino_r50_4scale_12ep_48.87.txt))
...
[11/17 16:05:09] fvcore.common.checkpoint INFO: [Checkpointer] Loading from detectron2://ImageNetPretrained/torchvision/R-50.pkl ...

**Comment by jihwanp** [comment]
> > hello, I cant reproduce dino with R101 backbone. I only got 47.15 total AP without same config.
> > https://github.com/IDEA-Research/detrex/blob/main/projects/dino/configs/dino_r101_4scale_12ep.py
> 
> You can refer to this [tutorial](https://detrex.readthedocs.io/en/latest/tutorials/Converters.html#download-pretrained-weights) to see how to convert the R-101 pretrained weights from torchvision downloaded checkpoints.
> 
> And maybe you can try to train DINO-R101 under 8 GPUs, which means 2 batch size each GPU, as we observed, 1/2/4 GPUs training is not better than 8 GPUs @adamqian111

@rentainhe  Why is there a difference depending on the number of GPUs? And is the difference significant?
